src/cli/index.ts

```diff
@@ -273,39 +282,42 @@ async function main(argv: string[]): Promise<void> {
   // overwrite the schema.prisma file if there's no remaining operations
   console.log(
     redent(`
-      Congratulations! You've fully upgraded your Prisma schema.
-
-      As a last step, some final adjustments will be made to your Prisma 2.0 
-      schema to carry over some Prisma-level attributes that aren't picked 
-      up by introspection.
+      ${bold('➤ What happens next')}
+      As a last step, some final adjustments will be made to your Prisma 2 schema
+      to carry over some Prisma-level attributes that aren't picked up by introspection.
 
-      ${bold('Warning')}
-      Your current Prisma 2.0 schema will be overwritten, so please
-      make sure you have a backup!
+      ${bold('➤ Note')}
+      This overwrites your current Prisma 2 schema. In case you made any manual adjustments
+      to the Prisma schema (e.g. renamed a relation field), you might want to back up your
+      current schema file.
   `)
   )
   await confirm(`Are you ready? [Y/n] `)
   clear(true)
   console.log()
   const relpath = path.relative(process.cwd(), schemaPrisma)
-  console.log(`Updating ${relpath}...`)
+  console.log(`Updating ${bold(gray(relpath))}...`)
   await writeFile(schemaPrisma, schema.toString())
-  console.log(`Updated ${relpath}.`)
+  console.log(`Done updating ${bold(gray(relpath))}!`)
   console.log(
     redent(`
-      ✔ You're all set!
+      ${bold(`✔ Congratulations, you're all set!`)}
+
+      ${bold('➤ Note')}
+      If you didn't execute all generated SQL commands against your database, 
+      you can re-run the Upgrade CLI at any time. 
 
-      ${bold('Notes')}
-      The Upgrade CLI doesn't resolve all of the schema incompatibilities.
-      If you want to resolve the remaining ones, you can follow this guide:
-      https://pris.ly/d/upgrading-the-prisma-layer
+      Note that the Upgrade CLI doesn't resolve all of the schema incompatibilities
+      between Prisma 1 and Prisma 2. If you want to resolve the remaining ones, 
+      you can do so manually by following this guide:
+      ${gray(underline(`https://pris.ly/d/upgrading-the-prisma-layer`))}
 
-      ${bold('Next steps')}
-      You can continue your upgrade process by installing Prisma Client 2.0:
-      npm install @prisma/client
+      ${bold('➤ Next steps')}
+      Otherwise you can continue your upgrade process by installing Prisma Client 2:
+      ${green(`npm install @prisma/client`)}
```

Comment:
This should not really be the next step, right? Just calling `npx prisma generate` should take care of that automagically.

Reply:
Doesn't `npm install @prisma/client` also invoke `prisma generate`? For the first invocation, I typically prefer showing the `npm install @prisma/client` command since that's the familiar workflow for Node devs...